dts/bindings/dac/ti,dac161s997.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2025 Vitrolife A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI DAC161S997 16-bit 1 channel SPI DAC for 4-20 mA loops
+
+compatible: "ti,dac161s997"
+
+include: [dac-controller.yaml, spi-device.yaml]
+
+properties:
+  errb-gpios:
+    type: phandle-array
+    description: DAC error signal output.
```

Comment:
A more elaborate description would be nice.

Reply:
Added a bit more description. Not sure how much is needed since a lot is documented in the header.